modules/persistent-storage-csi-migration-vsphere-to-4-13.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-vsphere-to-4.13_{context}"]
+= Updating from {product-title} 4.12 to 4.13 with vSphere in-tree PVs
+
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.12 to 4.13, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} update can occur and automatic CSI migration for vSphere occurs only if you opt in.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by running the following command to perform an administrator acknowledgment:
+
+[source, terminal]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-126-vsphere-migration-in-4.14":"true"}}' --type=merge
+----
```

Comment:
I am wondering if we should add a paragraph below that, it is generally safe to provide this ack because migration is obviously not enabled in 4.13. So something along the line of:

"It should be generally okay to provide requested admin-ack for upgrades from 4.12 to 4.13 because CSI migration is not yet enabled for upgraded clusters from 4.12 to 4.13. But we recommend that you start planning an upgrade of your vSphere environment for future upgrade to 4.14, so as all the in-tree volumes can be managed by CSI seamlessly"

Reply:
I think that's what Lisa mentioned below. Maybe it can be reworded in a more positive fashion

Reply:
Yeah I think section below doesn't say, why it is okay to provide admin-ack in 4.12 for upgrade to 4.13. The section below is all about, why you shouldn't provide admin-ack.  I am trying to say, why it is generally safe to provide admin-ack for 4.12 to 4.13 upgrades at least.

Reply:
Done.